src/kbmod/search/KBMOSearch.cpp

```diff
@@ -106,7 +106,7 @@ void KBMOSearch::setStartBoundsY(int y_min, int y_max) {
 }
 
 void KBMOSearch::search(int aSteps, int vSteps, float minAngle, float maxAngle, float minVelocity,
```

Comment:
`minObservations` --> `min_observations` but `aSteps`, `vSteps`, `minAngle` etc..?

Reply:
I fixed this (here and throughout the parameter definitions)